doc/source/myrocks/zenfs.rst

```diff
@@ -30,8 +30,8 @@ The following procedure installs Percona Server for MySQL and then configures
 
 .. note::
 
-   The ``<block_device_name>`` can have a short name designation which is the ``<short_block_device_name>``. For the purposes of this document, the ``block_device_name`` is ``/dev/nvme0n2`` and the short name is ``nvme0n2``. 
-    
+   The ``<block_device_name>`` can have a short name designation which is the ``<short_block_device_name>``. For example, if the ``<block_device_name>`` is ``/dev/nvme0n2`` remove the ``/dev/`` portion and the ``<short_block_device_name>`` is ``nvme0n2``.
```

Comment:
remove the marks from <block_device_name> since this is a substitution.
There are several of these places where we should remove the marks.